.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Please delete the `/.idea/` folder and contents (**not** `/editor/.idea/`). Make sure you open up the editor folder in Android Studio, not the root soil-topArgraphy folder.